packages/litexa/src/parser/skill.coffee

```diff
@@ -173,7 +172,7 @@ class lib.Skill
     @tests = {}
     @dataTables = {}
     @sayMapping = {}
-    @dbTypes = {}
+    @dbTypes = []
```

Comment:
Rather than a flat array, I'd consider keeping this a map and add languages as the top level key.

```
dbTypes: {
  default: {
    dbTypeName: dbTypeClass,
    dbTypeName: dbTypeClass
  }
  language: {
    ...
  }
}
```